### PR TITLE
SD-5316 Pressing enter in the body changes the font for the text after the cursor

### PR DESCRIPTION
**Pull request by lnogues**
This issue seems to be directly related to contenteditable. So for now, the only way I found to modify it, is to manually remove any styling attributes in any span elements.

### CODE REVIEW

File: scripts/superdesk/editor2/editor.js [hunk 1]
```diff
@@ -1020,7 +1020,10 @@ angular.module('superdesk.editor2', [
                         scope.medium.subscribe(eventName, updateAddContentButton);
                     });
                     // listen updates by medium editor to update the model
-                    scope.medium.subscribe('editableInput', function() {
+                    scope.medium.subscribe('editableInput', function(e, elem) {
+                        elem.querySelectorAll('span[style]')
+                            .forEach(span => span.removeAttribute('style'));
```
Comment (petrjasek) [inline scripts/superdesk/editor2/editor.js]: would it be possible to remove those spans altogether?

Comment (lnogues) [inline scripts/superdesk/editor2/editor.js]: Sure, unless there are some specific usage for them?

Comment (petrjasek) [inline scripts/superdesk/editor2/editor.js]: I don't think so, it's just added by chrome, in firefox it shoudn't be there at all

Comment (lnogues) [inline scripts/superdesk/editor2/editor.js]: I can remove it

**Comment by petrjasek** [comment]
:+1:

**Pull request merged**